### PR TITLE
Feature/ch169/create patient on boarding functionality

### PR DESCRIPTION
**Pull request by fahadhussain2**

### CODE REVIEW

File: packages/app/src/containers/SearchConditions/selectedCondition.tsx [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+            <React.Fragment>
+                <Header
+                    title={'CONDITIONS'}
+                    leftIcon={true}
+                    rightIcon={true}
+                    navigation={this.props.navigation} />
+                <View style={styles.container}>
+                    <View style={styles.topView}>
+                        <Text style={styles.heading}>{Conditions}</Text>
+                        <Text style={styles.subheading}>{"This condition has been added to your journal. Do you suufer from any other conditions?"}</Text>
```
Comment (petermsouzajr) [inline packages/app/src/containers/SearchConditions/selectedCondition.tsx:45]: 'suffer' is misspelled

**Review by petermsouzajr — CHANGES_REQUESTED** [review]
small spelling error. rebase with master after branch ch168 is merged

**Review by petermsouzajr — CHANGES_REQUESTED** [review]
please resolve conflicts and spelling error

File: packages/app/src/containers/SearchConditions/selectedCondition.tsx (hunk 1 above)
Comment (fahadhussain2) [reply packages/app/src/containers/SearchConditions/selectedCondition.tsx:45]: @petermsouzajr Ok I will fix this typo error

**Comment by fahadhussain2** [comment]
> please resolve conflicts and spelling error

ok @petermsouzajr

**Comment by fahadhussain2** [comment]
@petermsouzajr All the conflicts are resolved now.

**Review by petermsouzajr — APPROVED** [review]
👍

**Pull request merged**